config/example.yaml

```diff
@@ -30,6 +30,8 @@ grains:
     absolute:
       - 1
       - -1
+dnatracing:
+  run: true # Whether to run DNA Tracing. Options : true, false
 plotting:
   save: True
```

Comment:
Does `read_yaml` read true/false as Python bool True/False? if so should we make these the same so they are consistent?

Reply:
Depends on the [version of YAML](https://stackoverflow.com/questions/42283732/are-on-and-off-supposed-to-be-interpreted-as-true-or-false-in-yaml-1-2), out of habit I've always used `true` as that seems to be what 1.2 uses. I'll update them all